CONTRIBUTING.md

```diff
@@ -74,6 +74,32 @@ environment variable to disable webpack optimizations.
 env DEBUG=true yarn start:prod
 ```
 
+## Authentication in your local Force app
+
+Authentication in Force is handled through OAuth, with [Gravity](https://github.com/artsy/gravity) authenticating the user and redirecting back to Force. For security reasons, the `localhost` origin [is forbidden as a redirect URL by Gravity in the staging environment](https://github.com/artsy/gravity/blob/543373d7d4413f5c8b1c8f84f73b2a592c00cba2/app/models/util/url_validation.rb#L23). This means that when running Force locally at `http://localhost:5000`, you won't be able to sign up or log in.
```

Comment:
This isn't totally true -- the _redirect_ will fail to redirect back to localhost but a user can login and create just fine -- they just have to manually go back to localhost:5000

Reply:
Wow, I had no idea refreshing would work. I'll think about how to rewrite this, because I think it's still valuable to define a flow that doesn't include refreshing after a failed redirect.